### PR TITLE
Feat: add boots table

### LINKED ISSUE #145: show execution items (table) for Boots

### PR DESCRIPTION
**Pull request by mari1912**
- closes #145 
- update query to get information from boots and without boots] 
- add boots details table in boots tab
image

### CODE REVIEW

File: backend/kernelCI_app/urls.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
     ),
     path("build/<str:build_id>", views.BuildDetails.as_view(), name="buildDetails"),
     path("build/<str:build_id>/tests", views.BuildTests.as_view(), name="buildTests"),
-    path("tests/<str:test_id>", views.TestDetails.as_view(), name="testDetails"),
-    path("improv/<str:commit_hash>", views.TestsByCommitHash.as_view(), name="testsByCommitHash"),
+    path("tests/test/<str:test_id>", views.TestDetails.as_view(), name="testDetails"),
+    path("tests/<str:commit_hash>", views.TestsByCommitHash.as_view(), name="testsByCommitHash"),
```
Comment (WilsonNet) [inline backend/kernelCI_app/urls.py:15]: testsByTreeAndCommitHash

File: backend/kernelCI_app/views/testsByCommitHashView.py [hunk 2]
```diff
@@ -39,7 +45,8 @@ def get(self, request, commit_hash: str | None):
                 t.environment_comment, t.misc FROM checkouts AS c
                 INNER JOIN builds AS b ON c.id = b.checkout_id
                 INNER JOIN tests AS t ON t.build_id = b.id
-                WHERE c.git_commit_hash = %s {path_filter}
+                WHERE c.git_commit_hash = %s AND c.origin = %s AND c.git_repository_url = %s
+                  AND c.git_repository_branch = %s {path_filter}
                 ORDER BY
                 build_id,
```
Comment (WilsonNet) [inline backend/kernelCI_app/views/testsByCommitHashView.py:51]: we should order by path here

**Review by WilsonNet — COMMENTED** [review]
great job, almost there

File: backend/kernelCI_app/views/testsByCommitHashView.py (hunk 2 above)
Comment (mari1912) [reply backend/kernelCI_app/views/testsByCommitHashView.py:51]: yeah, I realize it when I did in #171, but I will adjust it here

**Review by WilsonNet — APPROVED** [review]
Nice job, we will need a ticket for the status filter in this table
and it should be 6 filters (probably a select)

**Pull request merged**